libs/rtemodel/include/RteTarget.h

```diff
@@ -853,6 +853,17 @@ class RteTarget : public RteItem
   */
   RteDeviceProperty* GetDeviceEnvironment() const { return m_deviceEnvironment; }
 
+  /**
+   * @brief get the generator input file path
+   * @return generator input file
+  */
+  const std::string& GetGeneratorInputFilePath() const { return m_generatorInputFilePath; }
```

Comment:
See previous comment: rename method to GetGeneratorInputFile() if it returns filename.